.github/CONTRIBUTING.md

```diff
@@ -8,12 +8,12 @@ get involved:
   Select `New issue`, fill in a descriptive title, and provide as much detail 
   as is needed for the issue to be reproduced.
 
-* **Notebooks**: If you would like to contribute a notebook, please 
-  create a [fork](https://help.github.com/articles/fork-a-repo/) of the repository 
-  from the `master` branch and create a 
+* **Notebooks**: If you would like to contribute a notebook, please do it in [Qiskit Community Tutorials](https://github.com/qiskit-community/qiskit-community-tutorials).
+  Create a [fork](https://help.github.com/articles/fork-a-repo/) of the repository
+  from the `master` branch and create a
   [pull request](https://help.github.com/articles/about-pull-requests) for your change.
-  Note that new notebooks should be placed in the relevant part of the 
-  [Community](./community/) section.
+  Note that new notebooks should be placed in the relevant part of the
```

Comment:
I would remove this sentence altogether. It's pointing at entire workspace made up of (currently) 32 repos. If I were reading this trying to figure out how to contribute a new notebook, and I'm not doing it in `qiskit-community-tutorials`, then this link definitely isn't going to clarify it for me.